### PR TITLE
fix(dav): Always respond custom error page on exceptions

### PR DESCRIPTION
**Pull request by artonge**
In order to match what we are doing with regular controlers.

### CODE REVIEW

File: apps/dav/lib/Files/ErrorPagePlugin.php [hunk 1]
```diff
@@ -98,4 +94,14 @@ public function sendResponse() {
 		$this->server->sapi->sendResponse($this->server->httpResponse);
 		exit();
 	}
+
+	private function acceptHtml(): bool {
+		foreach (explode(',', $this->request->getHeader('Accept')) as $part) {
+			$subparts = explode(';', $part);
+			if (str_ends_with($subparts[0], '/html')) {
+				return true;
+			}
```
Comment (provokateurin) [inline apps/dav/lib/Files/ErrorPagePlugin.php:103]: IMO checking for the full mimetype `text/html` is better as HTML only has this single mimetype that is used, but this should work as well :woman_shrugging:

**Review by provokateurin — APPROVED** [review]

File: apps/dav/lib/Files/ErrorPagePlugin.php (hunk 1 above)
Comment (artonge) [reply apps/dav/lib/Files/ErrorPagePlugin.php:103]: Let's keep it like that :)

**Review by AndyScherzinger — APPROVED** [review]

**Pull request merged**